packages/esm-patient-chart-app/src/clinical-views/utils/encounter-list-config-builder.ts

```diff
@@ -42,7 +48,7 @@ const getColumnValue = (encounter: Encounter, column: ColumnDefinition, config:
   }
 
   if (column.valueMappings) {
-    return resolveValueUsingMappings(encounter, column.concept, column.valueMappings);
+    return resolveValueUsingMappings(encounter, column.concept, column.valueMappings, t);
   }
```

Comment:
So the last thing I think I'd want to add here is something like:

```ts
const resolvedValue = resolveValueUsingMappings(encounter, column.concept, column.valueMappings, t);
if (resolvedValue !== null && resolvedValue !== undefined) {
  return resolvedValue;
}
```

The idea here is to allow a fall-back for when a column has `valueMappings` but not `valueMapping` specifically for the concept in this column. In those cases, it should be handled by the `return getObsFromEncounter()` below.